src/types/tracker.ts

```diff
@@ -67,22 +106,28 @@ export type TrackedEnvironmentData = {
   browser_version: string
 }
 
+export type AnalyticsEventProperties = {
+  event_type?: TrackedEventTypes
+  step?: string
+  is_cross_device?: boolean
+  is_custom_ui?: boolean
+  status?: string
+  capture_method_rendered?: 'upload' | 'camera'
+  document_side?: 'front' | 'back'
+  video_capture_step?: 'step1' | 'step2'
+  link_method_selected?: 'copy' | 'qr_code' | 'sms'
```

Comment:
the TrackedEventTypes is exported but the rest (i.e. capture_method_rendered, document_side) are not. This seems to be inconsistent.

I know we'll get rid of these anyhow so its a minor comment.

Reply:
Good point, I was using these types in another file but then I removed it. No need to export `TrackedEventTypes`. 
Some properties are string literal types because we know what the possible values will be, in other cases we don't.
I hope that answer your question

Reply:
yeah fair enough